### DISCUSSION TITLE
I have a query requirement.

### DISCUSSION THREAD

**Discussion by ddxl123** [simolus3/drift#2202] [Q&A] [answered]
There is a field in a table: `mul_times`, the value type is string, such as "3,4,5,6".

I want to find all lines where the number between the first comma and the second comma is greater than 5, 
and finally return an array of numbers between the first comma and the second comma.

How to do it?

**Comment by simolus3** [comment, marked as answer]
>  the value type is string, such as "3,4,5,6".

Do you control that data? It's a lot easier if you're using a json array (e.g. `[3,4,5,6]`).
With JSON, you could use

```dart
final secondValue = table.columnWithNumber.jsonExtract<int>(r'$[1]');

final query = selectOnly(table)
    ..addColumns([secondValue])
    ..where(secondValue.isBiggerOrEqualValue(5));
final list = await query.map((row) => row.read(secondValue)).toList();
```

**Reply by ddxl123** [reply]
Thanks!
